### DISCUSSION TITLE
Telemetry when Car Leaves Track

### DISCUSSION THREAD

**Discussion by RohanVDeshpande** [theOehrly/Fast-F1#309] [Q&A] [answered]
Is position telemetry data only captured when the car is _on the track_? 

I've been looking at races where drivers have left the track / crashed off of the track. I'm unable to find data showing that they left the track even though the video clearly shows the deviation.

A couple races that I've looked at:
* 2021 Abu Dhabi, HAM leaves the track on lap 1. I can't find this in the telemetry data
* 2022 Austin, BOT on lap 17 loses the back end of the car and spins out causing him to beach the car. I can see in the telemetry data that the car stops moving, but supposedly it stops moving *on the track* instead of spinning out.
Furthermore, in both cases, I would've expected the "Status" to show "OffTrack". However, it shows "OnTrack" the whole time.

**Comment by theOehrly** [comment, marked as answer]
The position data in the telemetry is kind of a "normalized" position data. There is only one line on which the car always is. Similar to how you cannot tell when a car leaves the track, you cannot tell apart different lines for a corner.
I have no idea how this is done in the backend. But the position data is not an actual GPS-like position.

**Reply by RohanVDeshpande** [reply]
Unfortunate :( . I wonder why they decided to do this... Analysis on the different racing lines / looking at when drivers went off track could've been really interesting.

**Reply by theOehrly** [reply]
They data is intended to be used for the visualization of a driver tracker (like the teams have on their pitwalls) to see the order of the cars on the track. It is sufficient for that. I guess on the side of F1, there is no need to provide more accurate data than that.
